docs/code-samples/settlement/daml/Scripts/Transfer.daml

```diff
@@ -1,43 +1,13 @@
 module Scripts.Transfer where
 
-import DA.Map qualified as M (empty, fromList)
-import DA.Set qualified as S (empty, fromList, singleton, fromList)
+import DA.Set qualified as S (fromList, singleton, fromList)
 import Daml.Script
 
--- INTERFACE DEPENDENCIES --
-import Daml.Finance.Interface.Account.Account qualified as Account (Controllers(..))
-import Daml.Finance.Interface.Account.Factory qualified as Account (F)
-import Daml.Finance.Interface.Holding.Factory qualified as Holding (F)
-import Daml.Finance.Interface.Holding.Transferable qualified as Transferable (I)
-import Daml.Finance.Interface.Instrument.Base.Instrument qualified as Instrument (I, getKey)
-import Daml.Finance.Interface.Types.Common.Types (Id(..))
+import Scripts.Setup
 
--- IMPLEMENTATION DEPENDENCIES --
-import Daml.Finance.Account.Account qualified as Account (Factory(..))
-import Daml.Finance.Holding.Fungible qualified as Fungible (Factory(..))
-import Daml.Finance.Instrument.Token.Instrument (Instrument(..))
-
-import Workflow.CreateAccount qualified as CreateAccount
-import Workflow.CreditAccount qualified as CreditAccount
 import Workflow.Transfer qualified as Transfer
 
--- | Outlines the required authorizers for an action.
-data RequiredAuthorizers
-  = Owner
-  | Custodian
-  | Both -- i.e., owner and the custodian
-  | Anyone
-  deriving (Eq, Show)
-
--- | Describes the required authorizers for incoming and outgoing transfers.
-data AccountControllers =
-  AccountControllers with
-    incoming : RequiredAuthorizers
-    outgoing : RequiredAuthorizers
-  deriving (Eq, Show)
-
--- | This is the type of data that the following script returns after setting up the request to
--- transfer a holding from Alice to Bob.
+-- | Return type of the setup script.
```

Comment:
Might be confused with the setup script defined in `Setup.daml`

Reply:
Thanks, I changed it to:

-- | Return type of the setup transfer request script.